Tools.Packager/Program.cs

```diff
@@ -163,84 +163,137 @@ public static async Task<int> Main(string[] args)
+                else if (Regex.IsMatch(opt.Version, "[0-9]+.[0-9]+.[0-9]+.[0-9]+$"))
                 {
-                    dmAppVersion = DMAppVersion.FromProtocolVersion(version);
+                    dmAppVersion = DMAppVersion.FromProtocolVersion(opt.Version);
                 }
                 else
                 {
                     // Supports pre-releases
-                    dmAppVersion = DMAppVersion.FromPreRelease(version);
+                    dmAppVersion = DMAppVersion.FromPreRelease(opt.Version);
                 }
             }
             else
             {
-                dmAppVersion = DMAppVersion.FromBuildNumber(buildNumber);
+                dmAppVersion = DMAppVersion.FromBuildNumber(opt.BuildNumber);
             }
 
-            if (String.IsNullOrWhiteSpace(packageName))
+            if (String.IsNullOrWhiteSpace(opt.PackageName))
             {
                 // Create default name if no custom name was used.
-                packageName = $"Package {dmAppVersion}";
+                opt.PackageName = $"Package {dmAppVersion}";
             }
 
-            switch (dmappType)
+            switch (opt.DmappType)
             {
                 case "automation":
                     appPackageCreator =
-                        AppPackageCreatorForAutomation.Factory.FromRepository(new Logging(debug), workspace, packageName, dmAppVersion);
+                        AppPackageCreatorForAutomation.Factory.FromRepository(new Logging(debug), opt.Workspace, opt.PackageName, dmAppVersion);
                     break;
 
                 case "visio":
                     appPackageCreator =
-                        AppPackageCreatorForVisio.Factory.FromRepository(new Logging(debug), workspace, packageName, dmAppVersion);
+                        AppPackageCreatorForVisio.Factory.FromRepository(new Logging(debug), opt.Workspace, opt.PackageName, dmAppVersion);
                     break;
 
                 case "protocolvisio":
                     appPackageCreator = AppPackageCreatorForProtocolVisio.Factory.FromRepository(FileSystem.Instance, new Logging(debug),
-                        workspace, packageName, dmAppVersion, protocolName);
+                        opt.Workspace, opt.PackageName, dmAppVersion, protocolName);
                     break;
 
                 case "dashboard":
                     appPackageCreator =
-                        AppPackageCreatorForDashboard.Factory.FromRepository(new Logging(debug), workspace, packageName, dmAppVersion);
+                        AppPackageCreatorForDashboard.Factory.FromRepository(new Logging(debug), opt.Workspace, opt.PackageName, dmAppVersion);
+                    break;
+
+                case "keystone":
+                    appPackageCreator =
+                        AppPackageCreatorForKeystone.Factory.FromRepository(metaData, FileSystem.Instance, new Logging(debug), opt.Workspace, opt.PackageName, dmAppVersion);
```

Comment:
![image](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.Packages/assets/97458010/6c3fd2bc-3f8f-4c0e-ba47-ba715390554e)

This is the 'Product version' of the exe (I took the bin folder of the tool itself):
![image](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.Packages/assets/97458010/607fe5c9-a950-4f23-a91b-b18434393c59)

Somehow, the creation of the nupkg removes that part? When specifying a specific version via command, it does work.